frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -533,6 +541,18 @@ export function Experiment_(): JSX.Element {
                                     >
                                         <b className="uppercase">{status()}</b>
                                     </Tag>
+                                    {experimentResults && experimentData.end_date && (
+                                        <Tag
+                                            style={{ alignSelf: 'center' }}
+                                            color={areResultsSignificant ? 'green' : 'geekblue'}
+                                        >
+                                            <b className="uppercase">
+                                                {areResultsSignificant
+                                                    ? 'Significant Results'
+                                                    : 'Results not significant'}
```

Comment:
```suggestion
                                                    : 'Results not yet significant'}
```

Reply:
This is shown on completion only, so shouldn't have "yet" in it.